lib/MetadataBuilder.js

```diff
@@ -1,4 +1,5 @@
 const fs = require('fs')
+const encodeURIComponent = require('strict-uri-encode')
```

Comment:
This is not exactly related to the issue but I thought it useful to actually produce correct templates. While I did not observe any problems, according to the URI Templates spec, template variables should also be encoded